matlab/src/cpp/arrow/matlab/error/error.h

```diff
@@ -173,4 +173,8 @@ namespace arrow::matlab::error {
     static const char* UKNOWN_TIME_UNIT_ERROR_ID = "arrow:matlab:UnknownTimeUnit";
     static const char* FIELD_FAILED_TO_CREATE_TYPE_PROXY = "arrow:field:FailedToCreateTypeProxy";
     static const char* ARRAY_FAILED_TO_CREATE_TYPE_PROXY = "arrow:array:FailedToCreateTypeProxy";
+    static const char* ARROW_TABULAR_SCHEMA_INVALID_NUMERIC_FIELD_INDEX = "arrow:tabular:schema:InvalidNumericFieldIndex";
+    static const char* ARROW_TABULAR_SCHEMA_INVALID_FIELD_NAME = "arrow:tabular:schema:InvalidFieldName";
```

Comment:
Could we rename this to `ARROW_TABULSR_SCHEMA_UNKNOWN_FIELD_NAME` to make it clear this error is about indexing.

Reply:
Thanks for the suggestion! I like `UnknownFieldName` more, as well.

Reply:
Fixed.